README.md

```diff
@@ -161,19 +161,21 @@ You can download the latest builds by following the links below:
 You can easily install the latest Homebrew edge build with the following command:
 
 ```bash
-brew install $(curl -Ls -o /dev/null -w %{url_effective} https://aka.ms/InstallAzureCliHomebrewEdge)
+# You need to uninstall the stable version with `brew uninstall azure-cli` first
+curl --location --silent --output azure-cli.rb https://aka.ms/InstallAzureCliHomebrewEdge
+brew install --build-from-source azure-cli.rb
```

Comment:
Use ` --build-from-source` to prevent brew installing pre-built binary packages.

Ref: https://docs.brew.sh/Manpage#install-options-formulacask-
> -s, --build-from-source: Compile formula from source even if a bottle is provided. Dependencies will still be installed from bottles if they are available.